### DISCUSSION TITLE
REPOSITORIO DE GITHUB DE BACKEND NO TIENE EL TEMARIO EDUCATIVO Y NINGUNA INFORMACION

### DISCUSSION THREAD

**Discussion by lopez-juandavid** [Launch-X-Latam/MisionFrontEnd#580] [Team Posts]
Muy buenas noches. 

ando bastante atrasado con el back end, en donde a día hoy apenas llegue a mirar el repositorio de gitub de back end a la que me uní desde un principio, y resulta que no hay ningun temario educativo dentro del repositorio que llegue a copiar. Que paso alli? Existen otro repositorio de back end que ha lanzado innovacion virtual?

este es el link del repositorio: https://github.com/Launch-X-Latam/MisionBackend

cabe aclarar que estoy dentro del grupo de backend de teams.

**Comment by Martscribe** [comment]
Estoy en las mismas ahorita

**Comment by DavidSantana872** [comment]
X3 estoy en las mismas y el teams me va super mal lentisimo

**Comment by lopez-juandavid** [comment]
Si, me hizo muy raro que eso este ocurriendo. Alguno sabría el correo de contacto de launch x para reportar el fallo?

**Comment by mariocanul30** [comment]
El sherpa dijo el día de hoy, que este fin de semana ya sube toda la información en el repositorio, puesto que hubo varios cambios al temario, ya que no acabamos la parte de frontend, y esta semana eso estuvimos viendo con el.

**Comment by Eduardo33q** [comment]
Tienen el link al grupo de teams?

**Comment by NelsonOrtizLeal** [comment]
Yo hice un fork antes de que quitara algunos temas. Puedes ver el repo que tenia en la copia que hice. Pero como dijo Mario puede que este cambie. 

https://github.com/NelsonOrtizLeal/MisionBackend

Saludo

**Comment by Amhedriel** [comment]
# Buenas noches compañeros 
Pero me temo que están un poco mal informados con respecto a ***LaunchX***, este modo de estudio no es una Universidad Online o uno de esos centros educativos Online como Platzi o EDTeam, aquí no pagas "mensualidad" para tener "clases" y "apuntes" con todo el "material" que quieres para comprender muy bien una materia en específico paso a paso en un solo lugar y sin más.

Con esto claro, lo que yo comprendí desde el primer video que hicieron acá en LaunchX, se trata de que los Sherpas (Guía o porteador en las expediciones en el Himalaya no profesor de escalado) nos guiarán por la ruta de aprendizaje del FrontEnd cómo del BackEnd dándonos las pautas, consejos, términos, temas, etc. que debemos conocer, pero, **tenemos que buscar por nosotros mismos la forma de aprenderlos de manera detallada a modo "AUTODIDACTA"** por eso aquí  no pasan asistencia, exámenes, revisión de apuntes, promedios ni nada de eso, ***eso ya depende de nosotros*** y solo de nosotros, no es cosa de culpar a los demás por nuestro descuido y falta de atención a este tipo de cosas. Revisen los videos y estudien los temas que dicen los guías, incluso dan direcciones de páginas para profundizar en esos conocimientos, de herramientas, incluso de otros guías y algunos compañeros que compartimos páginas de recursos gratuitos.

Espero que comprendan el modo de ayuda que nos dan aquí, no se olviden que lo único que pidieron es que las prácticas estén en nuestros repositorios personales y tratar de completarlos lo más rápido posible, también prepárense para el Hackatton, saludos.

**Comment by Leonardo-Ismet** [comment]
El repositorio si cuenta con la información, aunque no se está actualizando de forma automática y deben hacerlo de forma manual, para esto si no tienen el fork solo lo crean, ahora si ya tienen el fork, buscan la barra de notificaciones donde avisa sobre los commits, al final de esa barra aparece una pestaña denominada sync fork, presionan y luego presionan en el botón Update branch y se actualizara la información, hay que realizar esto con cada actualización, dejo una imagen de referencia, espero le ayude, saludos.
 
![actualizar fork](https://user-images.githubusercontent.com/88945000/204857720-bb559ea8-9a0b-435a-89ed-94db5bccf8c9.png)
